### PR TITLE
EKS Anywhere workshop

### PR DESCRIPTION
**Pull request by hariohmprasath**
*Issue #, if available:*
Workshop contents for EKS Anywhere service

*Description of changes:*
I restructured a few of the original docs to separate files to reuse as much as possible via `content` tag for this workshop, instead of repeating the content 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere/pull/648#" title="Author self-approved">hariohmprasath</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **rothgar**
You can assign the PR to them by writing `/assign @rothgar` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere).

<details open>
Needs approval from an approver in each of these files:

- **[docs/OWNERS](https://github.com/aws/eks-anywhere/blob/main/docs/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by abhay-krishna** [comment]
Hi @hariohmprasath. Thanks for your PR.

I'm waiting for a [aws](https://github.com/orgs/aws/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/aws/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by chrisnegus** [comment]
Thanks for providing this content @hariohmprasath . I think the idea of putting content you want to reuse into separate files is a really good idea. I won't have time to review this for a couple days, so I'm going to put this on hold for the moment.
/hold

**Comment by hariohmprasath** [comment]
> Thanks for providing this content @hariohmprasath . I think the idea of putting content you want to reuse into separate files is a really good idea. I won't have time to review this for a couple days, so I'm going to put this on hold for the moment. /hold

Thanks @chrisnegus, please take your time.

**Comment by rothgar** [comment]
An interesting side effect of using `{{< content >}}` shortcodes is we lose the section titles and links on the right nav bar. This PR is on the right in this picture and the live site is on the left 
image

File: docs/content/en/docs/tasks/cluster/cluster-verify.md
Comment (rothgar) [inline docs/content/en/docs/tasks/cluster/cluster-verify.md:51]: If commands are example output we don't include `bash` because the rendered site will add a copy button if bash is included. We don't want people copying the example output by mistake and trying to run it.

**Comment by rothgar** [comment]
I'm having a hard time to understand the value proposition of the workshop content. A majority of the content already exists in the docs and repeating the content has minimal benefits.

The main new content added are

* cost management
* eks connector
* storage
* gitops workload management
* observability
* istio

istio, kubecost, and eks connector were intentionally left out of the original docs because they're not not part of the OSS project or they're not supported by the EKS-A team.

Kubecost isn't accurate on-prem unless a user calculates their costs for servers/compute/storage/etc. Users are free to run it but not something we currently support in any way.

Istio isn't supported by EKS-A and we've been trying very hard to keep official docs related to software we do support. If istio is part of the docs then it's likely we'd also want to include linkerd, cilium service mesh, etc.

EKS connector currently links to the official docs because it's not an EKS-A feature but rather an EKS feature. We want to be very clear that 1) EKS connector requires an AWS account 2) Documentation is owned by the team that manages it. Pulling that content into EKS-A docs can be nice but we want to be very careful about being obvious to customers when they're looking at a paid feature (official AWS docs) and when they're looking at something free (eks-a OSS docs).

The storage and observability sections could be useful to be folded into the docs/tasks and would be a good addition. We don't have official support for prometheus/grafana but it's hard to operate a cluster without them and I think there's a good argument they should be included. Would probably want @jaxesn opinion on if it should be included.

gitops workload management I'm torn on. I don't think we want to recommend users use the same gitops controller for cluster management and workloads. Partially to avoid blast radius and partially because I don't think it'll work with our default deployment strategy using a management cluster. That section could be part of tasks/workload management but IMO it should also include information on deploying a dedicated workload gitops controller and not re-using the gitops controller used for cluster changes.

There were also some formatting issues in some of the workshop and original docs because of the `{{< content >}}` shortcode and some sections got removed or modified that would need a more detailed review. I'm curious what other people on the team think and will wait for their reviews too.

File: docs/content/en/docs/getting-started/install/_index.md
Comment (rothgar) [inline docs/content/en/docs/getting-started/install/_index.md:87]: We shouldn't remove this section from the original doc without replacing it because it's useful for our standard docs flow.

**Review by chrisnegus — CHANGES_REQUESTED** [review]
With the suggested changes, I think we could merge this PR and continue to test and improve it. I would like to see content added to explain what the workshop is about and how we expect people to use the flow. Because I think there might be come confusion with so much duplicated content.

In general, however, well done! A lot of work went into this and I think it's generally an added value.

File: docs/content/en/docs/getting-started/install/_index.md
Comment (jonahjon) [reply docs/content/en/docs/getting-started/install/_index.md:27]: Changing to recommended verbage.

File: docs/content/en/docs/workshops/_index.md
Comment (jonahjon) [reply docs/content/en/docs/workshops/_index.md:10]: Changing to recommended verbage.

File: docs/content/en/docs/getting-started/install/_index.md
Comment (jonahjon) [reply docs/content/en/docs/getting-started/install/_index.md:27]: Actually this can be whole addition can be removed from this PR, it's not needed for the workshop content.

File: docs/content/en/docs/tasks/cluster/cluster-verify.md
Comment (jonahjon) [reply docs/content/en/docs/tasks/cluster/cluster-verify.md:32]: Changing to recommended.

File: docs/content/en/docs/workshops/clustermanagement/eksconnector/verificationandcleanup.md
Comment (jonahjon) [reply docs/content/en/docs/workshops/clustermanagement/eksconnector/verificationandcleanup.md:16]: Here we can see an  overview of the workloads in our cluster, showing metadata information such as `Name`, `Namespace`, and `Status`. 

We also the ability to filter by propeties such as `Namespace` and other searchable values such as `capi` to bring up all of our Cluster API workloads.

Additionally if we wanted more information of a particular workload we can click on the name of the deployment to get a more detailed view.

File: docs/content/en/docs/workshops/workloadmanagement/storage/localstorage.md
Comment (jonahjon) [reply docs/content/en/docs/workshops/workloadmanagement/storage/localstorage.md:15]: two spaces looks like it worked.

**Comment by jonahjon** [comment]
I'm going to be opening a new PR with just the workshop content, and all the edits called out in the comments. @hariohmprasath Hari has taken a new role and doesn't have time right now, so I'll be taking over on this helping getting it across the finish line and maintaining/updating.

**Comment by jaxesn** [comment]
Closing this one since @jonahjon took over in another PR

**Pull request closed without merging**
